applications/fishing-map/src/features/analysis/Analysis.tsx

```diff
@@ -163,20 +163,24 @@ function Analysis() {
   }
 
   const onDownloadClick = async () => {
-    const createReports: CreateReport[] = dataviews.map((dataview) => {
+    const reportDataviews = dataviews.map((dataview) => {
       const trackDatasets: Dataset[] = (dataview?.config?.datasets || [])
         .map((id: string) => dataview.datasets?.find((dataset) => dataset.id === id))
         .map((dataset: Dataset) => getRelatedDatasetByType(dataset, DatasetTypes.Tracks))
 
       return {
-        name: `${dataview.name} - ${t('common.report', 'Report')}`,
-        dateRange: staticTime as DateRange,
         filters: dataview.config?.filters || [],
         datasets: trackDatasets.map((dataset: Dataset) => dataset.id),
-        geometry: analysisGeometry as ReportGeometry,
       }
     })
-    await dispatch(createReportThunk(createReports))
+    const reportName = Array.from(new Set(dataviews.map((dataview) => dataview.name))).join(',')
+    const createReport: CreateReport = {
+      name: `${reportName} - ${t('common.report', 'Report')}`,
```

Comment:
In spanish we use Report {name} instead of {name} report, so what about using the [interpolation](https://www.i18next.com/translation-function/interpolation) and allow that?
Also the dataview names are not translated and we could use the dataset id as we do for the sidebar.
Another question is how far we want to go into this.. does it really worth it?

Reply:
The report name is not used in any place yet but it is required to create the report. I'll leave it as it is for now to keep it super simple, I don't think it worths investing more time on it at this point. Makes sense @j8seangel ?